src/main/java/zmaster587/advancedRocketry/client/render/planet/RenderPlanetarySky.java

```diff
@@ -479,7 +480,7 @@ else if(DimensionManager.getInstance().isDimensionCreated(mc.theWorld.provider.d
 					GL11.glRotatef(phaseInc, 0, 1, 0);
 					GL11.glPushMatrix();
 					
-					GL11.glRotatef(subStar.getStarSeperation()*(202-solarOrbitalDistance)/100f, 1, 0, 0);
+					GL11.glRotatef(subStar.getStarSeperation()*AstronomicalBodyHelper.getBodySizeMultiplier(solarOrbitalDistance), 1, 0, 0);
```

Comment:
There should probably be a small constant added here, that's what the 200 + 2 was intended to do originally.

Reply:
The reason I did not have a constant was that it has no reason to have one. I saw the constant as a way to have a star with non-zero size at 200. But, that's not needed with the new system.

Reply:
Im a derp, I should stop trying to math at 3am